tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -135,58 +135,45 @@ func runTraffic(ctx context.Context, lg *zap.Logger, tf traffic.Traffic, hosts [
 }
 
 func simulateTraffic(ctx context.Context, tf traffic.Traffic, hosts []string, ids identity.Provider, baseTime time.Time, duration time.Duration) []report.ClientReport {
```

Comment:
Use it in robustness tests too?

Reply:
Do you want that to be in a separate PR instead of this one?

Reply:
never mind, it's actually simper than I thought...

Reply:
Right, I would want to avoid one asymmetrically applying improvements to framework. It would make the codebase harder to maintain. As long as change is not huge we should apply it to both.